.changeset/healthy-chicken-tease.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
sorry, what is this?

Reply:
@philcchen 
Thanks for the review.

Changeset have been introduced recently.
It is used to publish SDK packages with auto-increment on semver.
https://github.com/orca-so/whirlpools?tab=readme-ov-file#changesets

This change doesn't affect client-side code, so I executed `yarn changeset --empty` and this file is generated. 🙂